Sources/Common/Store/GlobalDataStore.swift

```diff
@@ -2,11 +2,6 @@ import Foundation
 
 /// SDK data that is common between all site ids.
 public protocol GlobalDataStore: AutoMockable {
-    // site id used for the singleton instance of the SDK.
-    var sharedInstanceSiteId: String? { get set }
-    func appendSiteId(_ siteId: String)
-    // all site ids that have ever been registered with the SDK
-    var siteIds: [String] { get }
```

Comment:
This code was originally added to the SDK so we could do something such as running background queue tasks for *all* of the site-ids *ever* registered to the SDK. 

However, we have never used this feature so I removed it. We can bring it back in the future if we decide we need this feature.